File: app/src/main/java/com/riders/thelab/data/local/bean/UrlSuffix.java
package com.riders.thelab.data.local.bean;

public enum UrlSuffix {

    MOVIES_URL_SUFFIX("/json/glide.json");

    private final String value;

    UrlSuffix(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: core/data/src/main/java/com/riders/thelab/core/data/local/bean/UrlSuffix.kt
package com.riders.thelab.core.data.local.bean

enum class UrlSuffix(val value: String) {
    MOVIES_URL_SUFFIX("/json/glide.json");

    companion object {
        fun getValue(value: UrlSuffix): String = entries.first { it.name == value.name }.value
    }
}